Tzkt.Api/Websocket/Parameters/AccountsParameter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Tzkt.Api.Websocket
+{
+    public class AccountsParameter
+    {
+        public List<string> Addresses { get; set; }
+
+        public void EnsureValid()
+        {
+            if (!Addresses.Any())
```

Comment:
If `Addresses` is `null` you will get an null ref exception

Reply:
Fixed, thanks!